MobileFoodSchedules/MobileFoodSchedules/Pages/Index.cshtml.cs

```diff
@@ -5,8 +5,8 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
-using QuickType;
-using QuickTypee;
+using Permits;
```

Comment:
Permist and Schedules is more self-documenting than QuickType and QuickTypee.   This is a good change.